index.html

```diff
@@ -209,12 +213,24 @@ <h2>Processing</h2>
       <p>The user agent SHOULD abort the current request if the `href`
       attribute of the <a>link</a> element of a <a>preload link</a> is changed,
       removed, or its value is set to an empty string.</p>
-      <p>At these times, the user agent must <a>obtain the resource</a> given by the
-      link element.</p>
+      <p>At these times, the user agent must <a>fetch and process the linked resource</a>
+      given by the link element.</p>
       <p>Obtaining the resource given by a <a>preload link</a> element MUST NOT
       <a>delay the load event</a> of the element's <a>node document</a>.</p>
-      <p>Once a <dfn>preload resource has been obtained</dfn>, the user agent
-      must add request to fetch group's response cache.</p>
+      <p>Once a preload resource <a data-cite="!HTML#process-the-linked-resource">has been processed</a>,
+      the user agent must add the response to the fetch group's response cache. The user agent
```

Comment:
I'm not sure when the "fetch group's response cache" phrasing was added, but I don't think it's a thing nowadays...
@igrigorik - do you happen to remember?

Maybe it would be better to make it explicitly vague in the meantime until we define a proper preload cache, and just call it "cache"?

Reply:
I'd be ok with that, though I think I'd prefer the term "preload cache" since I feel it'd be good to distinguish it from the HTTP cache while the preload cache isn't fully specified. Do you have a preference? (Regardless, yeah I'm not sure what "fetch group's response cache is").

Reply:
"preload cache" SGTM!